pkgs/development/ocaml-modules/ounit2/default.nix

```
@@ -4,13 +4,13 @@ buildDunePackage rec {
   minimalOCamlVersion = "4.04";
 
   pname = "ounit2";
-  version = "2.2.6";
+  version = "2.2.7";
 
-  duneVersion = if lib.versionAtLeast ocaml.version "4.08" then "2" else "1";
+  duneVersion = "3";
```

```suggestion
  duneVersion = if lib.versionAtLeast ocaml.version "4.08" then "3" else "1";
```

The condition is there for a reason…